modules/templates/helper.go

```diff
@@ -795,6 +796,14 @@ func RenderCommitBody(ctx context.Context, msg, urlPrefix string, metas map[stri
 	return template.HTML(renderedMessage)
 }
 
+// Match text that is between back ticks. 
+var codeMatcher = regexp.MustCompile("`([^`]+)`")
+
+func RenderCodeBlock(htmlEscapedTextToRender template.HTML) template.HTML {
+	htmlWithCodeTags := codeMatcher.ReplaceAllString(string(htmlEscapedTextToRender), "<code>$1</code>") // replace with HTML <code> tags
```

Comment:
Also, I'm wondering if we should also allow the other basic markdown functions `**bold**`, `__italic__`, and `~~strikethrough~~`.
But there is no need to do that now.
However, when we do that, we most likely solve the problem I mentioned above as well.
(Ah, GitHub does not do that as well)

Reply:
Yup, so I'd like to keep it simple. As long as we do not consider it as a full-featured markdown, then there is no bug. 😁